src/Microsoft.DocAsCode.Dfm/DfmJsonTokenTreeRender.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm
+{
+    using Microsoft.DocAsCode.MarkdownLite;
+
+    public class DfmJsonTokenTreeRender : JsonTokenTreeRenderer
```

Comment:
Yes ,it will be added later!

---
In reply to: [77628063](https://github.com/dotnet/docfx/pull/667#discussion_r77628063) [](ancestors = 77628063)